app/models/page.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: pages
+#
+#  id           :integer          not null, primary key
+#  page_type    :integer
+#  title        :string
+#  content_html :string
+#  content      :string
+#  source_url   :string
+#  status       :integer
+#  screenshot   :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  company_id   :integer
+#
+
+class Page < ApplicationRecord
+  STATUSES = %i[active pending finished].freeze
```

Comment:
every value MUST be as constant too.
And you need to use this constants in all places in code